presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -70,6 +71,8 @@ public void testAggregateSingleColumn()
         assertQuery("SELECT MAX(comment) FROM orders");
     }
 
+    protected abstract TestTable createTableWithDefaultColumns();
```

Comment:
move this right below `testInsertForDefaultColumn` so that these two tightly related methods stay together